src/dataset/domain/models/DatasetPaginationInfo.ts

```diff
@@ -0,0 +1,7 @@
+import { PaginationInfo } from '../../../shared/domain/models/PaginationInfo'
+
+export class DatasetPaginationInfo extends PaginationInfo<DatasetPaginationInfo> {
+  constructor(page = 1, pageSize = 10, totalItems = 0, itemName = 'Dataset') {
```

Comment:
I understand that the values are intentionally hardcoded and that in future iterations pagination will be introduced as in files tab?